source/agora/consensus/Ledger.d

```diff
@@ -1295,22 +1326,24 @@ public class Ledger
     }
 }
 
-private Height lastPaymentBlock(in Height height, uint payout_period) @safe @nogc nothrow pure
+// This is the last block height that has had fees and rewards paid before the current block
```

Comment:
```suggestion
/// This is the last block height that has had fees and rewards paid before the current block
```

Reply:
Wasn't sure with `private` (or `unittest`) comments whether to use `///` or '//'.

Reply:
On unittests, use `//` most of the time, unless the unittest is *documenting* the previous function. Most of our unittests are testing the function (hence they are exposing / checking more of the internals), and don't qualify as documenting unittests.

On function, no matter the visibility, prefer `///`.